modin/pandas/series.py

```diff
@@ -149,6 +149,7 @@ def __init__(
                 )
             )._query_compiler
         self._query_compiler = query_compiler.columnarize()
+        self._query_compiler._shape_hint = "column"
```

Comment:
`.columnarize()` already sets the shape hint:
https://github.com/modin-project/modin/blob/cbb3b5da2cff8db2ff41715f1e98ae61d632eba1/modin/core/storage_formats/base/query_compiler.py#L1262
```suggestion
```

Reply:
These line trigger metadata access we want to avoid so we have to set shape_hint="column" beforehand.
https://github.com/modin-project/modin/blob/cbb3b5da2cff8db2ff41715f1e98ae61d632eba1/modin/core/storage_formats/base/query_compiler.py#L1258-L1260

Reply:
We can probably remove this line. I forgot to provide a hint to astype.